README.md

```diff
@@ -103,6 +104,16 @@ A few examples:
     vrack: "{{ vrackid }}"
 ```
 
+### Move IP to a given host
+
+```yaml
+- name: Add server to vrack
```

Comment:
I guess the comment here is wrong :)

Reply:
oopsy !! you right sorry let me fix this :rocket: